public/pages/AnomalyCharts/containers/FeatureBreakDown.tsx

```diff
@@ -0,0 +1,111 @@
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { get } from 'lodash';
+import { EuiFlexItem, EuiFlexGroup, EuiTitle, EuiSpacer } from '@elastic/eui';
+import { FeatureChart } from '../components/FeatureChart/FeatureChart';
+import {
+  Detector,
+  FeatureAttributes,
+  AnomalyPreview,
+  DateRange,
+  FEATURE_TYPE,
+} from '../../../models/interfaces';
+import { NoFeaturePrompt } from '../components/FeatureChart/NoFeaturePrompt';
+import { focusOnFeatureAccordion } from '../../EditFeatures/utils/helpers';
+
+interface FeatureBreakDownProps {
+  title?: string;
+  detector: Detector;
+  anomaliesResult: AnomalyPreview;
+  annotations: any[];
+  isLoading: boolean;
+  dateRange: DateRange;
+  featureDataSeriesName: string;
+}
+
+export const FeatureBreakDown = React.memo((props: FeatureBreakDownProps) => {
+  return (
+    <React.Fragment>
+      {props.title ? (
+        <EuiFlexGroup alignItems="flexEnd">
+          <EuiFlexItem>
+            <EuiTitle size="s" className="preview-title">
+              <h4>{props.title}</h4>
+            </EuiTitle>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ) : null}
+
+      <EuiSpacer size="s" />
+      {get(props, 'detector.featureAttributes', []).map(
+        (feature: FeatureAttributes, index: number) => (
+          <React.Fragment key={`${feature.featureName}-${feature.featureId}`}>
+            <FeatureChart
+              feature={feature}
+              featureData={get(
+                props,
+                `anomaliesResult.featureData.${feature.featureId}`,
+                []
+              )}
+              annotations={props.annotations}
+              isLoading={props.isLoading}
+              dateRange={props.dateRange}
+              featureType={
+                get(
+                  props,
+                  `detector.uiMetadata.features.${feature.featureName}.featureType`
+                ) as FEATURE_TYPE
+              }
+              field={
+                get(
+                  props,
+                  `detector.uiMetadata.features.${feature.featureName}.featureType`
+                ) === FEATURE_TYPE.SIMPLE
+                  ? get(
+                      props,
+                      `detector.uiMetadata.features.${feature.featureName}.aggregationOf`
+                    )
+                  : undefined
+              }
+              aggregationMethod={
+                get(
+                  props,
+                  `detector.uiMetadata.features.${feature.featureName}.featureType`
+                ) === FEATURE_TYPE.SIMPLE
+                  ? get(
+                      props,
+                      `detector.uiMetadata.features.${feature.featureName}.aggregationBy`
+                    )
+                  : undefined
+              }
```

Comment:
Maybe move some of this logic to the top of this function / make local variables? It could also help prevent redundant `get()` calls for the `featureType`. Small thing, kind of a stylistic choice I guess